cmd/testnet-reset/main.go

```diff
@@ -47,11 +47,16 @@ func main() {
 	ctx := context.Background()
 	env.Parse()
 
-	cur := time.Now()
+	pst, err := time.LoadLocation("America/Los_Angeles")
+	if err != nil {
+		log.Fatal(err)
+	}
+	cur := time.Now().In(pst)
+	log.Println("current time:", cur)
```

Comment:
Yes. The largest interval allowed for the scheduler is one day. I want the log to reflect what date/time the program was run (to verify the filtering).